internals/core_enums.py

```diff
@@ -173,6 +173,17 @@
   ROLLOUT_OTHER: "(RARE) It's complicated or unusual",
 }
 
+ROLLOUT_FULL = 0,
```

Comment:
Please name these constants with the full field name to distinguish them from the constants on the lines above.  E.g., ROLLOUT_FULL -> ROLLOUT_STAGE_PLAN_FULL.